docs/guides/unity-catalog.md

```diff
@@ -1,12 +1,132 @@
 ---
-page_title: "Unity Catalog setup on AWS"
+page_title: "Unity Catalog set up on AWS"
 ---
 
+# Deploying pre-requisite resources and enabling Unity Catalog (AWS Preview)
+
+-> **Private Preview** This feature is in [Private Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access. 
+
+Databricks Unity Catalog brings fine-grained governance and security to lakehouse data using a familiar, open interface. Deploying the underlying cloud resources and Unity Catalog objects can be done with Terraform to simplify the set up and enable automation
+
+This guide assumes you have `databricks_account_username` and `databricks_account_password` for [https://accounts.cloud.databricks.com](https://accounts.cloud.databricks.com) and can find `databricks_account_id` in the bottom left corner of the page, once you're logged in. This guide is provided as-is and assumes you'll use it as the basis for your setup.
+
+Below are the high level steps to get started with Unity Catalog:
+- [Initialize the required providers](#provider-initialization)
+- [Configure AWS objects](#configure-aws-objects)
+  - A S3 bucket to store data from managed tables in Unity Catalog
+  - An IAM policy to define permissions to access data
+  - An IAM role that will be assumed by Unity Catalog to access data 
+- [Create users and groups who can access Unity Catalog](#create-users-and-groups)
+- [Create a Unity Catalog metastore and link it to workspaces](#create-a-unity-catalog-metastore)
+- [Create Unity Catalog objects in the metastore](#create-unity-catalog-objects)
+- [Configure external tables and credentials](#configure-external-tables)
+
+## Provider initialization
+
+> Initialize provider in "MWS" mode to set up account-level resources
```

Comment:
probably remove `>` and also explain/link to other pages to explain why we have to separate into different provider aliases.

Reply:
done